index.html

```diff
@@ -15,17 +17,129 @@
 </head>
 
 <body>
+  <div class="container">
+    <!-- nav panel  -->
+    <!-- <nav>
+        <div class="row d-flex justify-content-between">
+            <a class="btn btn-default" href="#">Home</a>
+            <a class="btn btn-default" href="#">About</a>
+            <a class="btn btn-default" href="#">Portfolio</a>
+            <a class="btn btn-default" href="#">Contact</a>
+        </div>
+      </nav> -->
+
+    <!-- navbar -->
+
+    <nav class="navbar navbar-light bg-light">
+      <a class="navbar-brand" href="#"><span class="color-me">Home</span></a>
+      <a class="navbar-brand" href="#"><span class="color-me">About</span></a>
+      <a class="navbar-brand" href="#"><span class="color-me">Portfolio</span></a>
+      <a class="navbar-brand" href="#"><span class="color-me">Contact</span></a>
+      <a href="https://lambdaschool.com/"><img src="lambda.jpg" width="50" height="50" alt="lambda-logo"></a>
```

Comment:
I recommend modifying height and width in CSS instead of in HTML, but this is fine too!